etna/datasets/internal_datasets.py

```diff
@@ -509,24 +518,26 @@ def get_m3_dataset(dataset_dir: Path, dataset_freq: str) -> None:
     TSDataset.to_dataset(df_test).to_csv(
         dataset_dir / f"m3_{dataset_freq.lower()}_test.csv.gz", index=True, compression="gzip", float_format="%.8f"
     )
-    TSDataset.to_dataset(df_full_exog).to_csv(
-        dataset_dir / EXOG_SUBDIRECTORY / f"m3_{dataset_freq.lower()}_full_exog.csv.gz",
-        index=True,
-        compression="gzip",
-        float_format="%.8f",
-    )
-    TSDataset.to_dataset(df_full_exog).to_csv(
-        dataset_dir / EXOG_SUBDIRECTORY / f"m3_{dataset_freq.lower()}_train_exog.csv.gz",
-        index=True,
-        compression="gzip",
-        float_format="%.8f",
-    )
-    TSDataset.to_dataset(df_test_exog).to_csv(
-        dataset_dir / EXOG_SUBDIRECTORY / f"m3_{dataset_freq.lower()}_test_exog.csv.gz",
-        index=True,
-        compression="gzip",
-        float_format="%.8f",
-    )
+
+    if dataset_freq != "other":
```

Comment:
Do I understand correctly, that for `other` we don't have `df_exog`?

Reply:
Yes, there are only integer timestamps. However, it is not better than our int timestamps.